lib/src/components/NxToggle/__tests__/NxToggle.test.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import NxToggle, { Props } from '../NxToggle';
+
+describe('NxToggle', function() {
+  const simpleProps: Props = {
+    toggleId: 'toggle-id',
+    isChecked: false,
+    onChange: () => {},
+    disabled: undefined,
+    children: undefined
+  };
+
+  function getShallowComponent(additionalProps?: Partial<Props>) {
```

Comment:
We have this function in a utility class now.  See the tests for virtually any other component for examples.

Reply:
> We have this function in a utility class now. See the tests for virtually any other component for examples.

Done here: https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/2fdd21bb12aab3f9b2d7bccc7aff2cf2212e3a3a